microx/Executor.cpp

```diff
@@ -1848,18 +1876,50 @@ static void ExecuteNative(void) {
 }
 
 static void ExecuteNativeAVX(void) {
+#ifdef _WIN32
+  gExceptionCode = EXCEPTION_ILLEGAL_INSTRUCTION;  // TODO(pag): Implement this!
+#else
   gSignal = SIGILL;  // TODO(pag): Implement this!
+#endif //_WIN32
 }
 
 static void ExecuteNativeAVX512(void) {
+#ifdef _WIN32
+  gExceptionCode = EXCEPTION_ILLEGAL_INSTRUCTION;  // TODO(pag): Implement this!
+#else
   gSignal = SIGILL;  // TODO(pag): Implement this!
+#endif //_WIN32
 }
 
+#ifdef _WIN32
+LONG WINAPI VectoredHandler(struct _EXCEPTION_POINTERS *ExceptionInfo) {
+#ifdef _WIN64
+#define Cip ExceptionInfo->ContextRecord->Rip
+#define Csp ExceptionInfo->ContextRecord->Rsp
+#else
+#define Cip ExceptionInfo->ContextRecord->Eip
+#define Csp ExceptionInfo->ContextRecord->Esp
+#endif //_WIN64
+  auto execArea = (uintptr_t)gExecArea;
+  if(Cip >= execArea && Cip < execArea + kPageSize) {
```

Comment:
There is a `.clang-format` config specifying Google style (that dictates space between `if` and `(`). It might be a good idea to run the code through the `clang-format` tool.

Reply:
There was no clang format when I implemented this, but feel free to run my code through the formatter and push to the branch.